Cargo.toml

```diff
@@ -16,11 +16,14 @@ exclude = ["/.travis.yml"]
 [badges]
 travis-ci = { repository = "nicokoch/reflink" }
 
+[dependencies]
+tempfile = "3.3.0"
```

Comment:
This afaik shouldn't be a dependency, what is the motivation for this?

Reply:
Oh I've already removed that.

Checkout the latest source code at [cargo-bins/reflink](https://github.com/cargo-bins/reflink/blob/main/Cargo.toml).